test/html_parser.test.js

```diff
@@ -710,6 +713,365 @@ test('Verifies that the CS degree audit is properly reproduced by the code', ()
+                        subject: 'CS',
+                        classId: '3000',
+                        credithours: '4.00',
+                        season: 'S2',
+                        year: '19',
+                        termId: '201960' 
+                    },
+                    { 
+                        hon: false,
+                        subject: 'CS',
+                        classId: '3500',
+                        credithours: '4.00',
+                        season: 'FL',
+                        year: '19',
+                        termId: '202010' 
+                    },
+                    { 
+                        hon: false,
+                        subject: 'CS',
+                        classId: '3650',
+                        credithours: '4.00',
+                        season: 'FL',
+                        year: '19',
+                        termId: '202010' 
+                    },
+                    { 
+                        hon: false,
+                        subject: 'CS',
+                        classId: '3800',
+                        credithours: '4.00',
+                        season: 'FL',
+                        year: '19',
+                        termId: '202010' 
+                    },
+                    { 
+                        hon: false,
+                        subject: 'MATH',
+                        classId: '2331',
+                        credithours: '4.00',
+                        season: 'S2',
+                        year: '19',
+                        termId: '201960' 
+                    },
+                    { 
+                        hon: true,
+                        subject: 'PHIL',
+                        classId: '1145',
+                        credithours: '4.00',
+                        season: 'FL',
+                        year: '19',
+                        termId: '202010' 
+                    } 
+                ],
+                "nupaths":[
+                    'SI',
+                    'ER' 
+                ]
+            },
+            "requirements":{
+                "classes": [ 
+                    { 
+                        subject: 'CS', 
+                        classId: '1210' 
+                    },
+                    { 
+                        subject: 'CS', 
+                        classId: '3700' 
+                    },
+                    { 
+                        subject: 'CS', 
+                        classId: '4400'
+                    },
+                    { 
+                        subject: 'CS', 
+                        classId: '4500' 
+                    },
+                    { 
+                        subject: 'THTR', 
+                        classId: '1170' 
+                    },
+                    { 
+                        subject: 'CS', 
+                        list: [ '4100', '4300', '4410','4150','4550','4991','4900'],
+                        num_required: '1'
```

Comment:
It would be cool if we had a "type" property here for the type of requirement needed. I'll open another issue. 

Ex. this one might have type "or" or "union", for ranges we could have type "range"

Reply:
I have another issue out for that - figure it should have been distinct.